### DISCUSSION TITLE
Need help with audio guide, how to use your library

### DISCUSSION THREAD

**Discussion by beatssQ** [google/oboe#1455] [General]
Hello, i am beginner programmer and it's to hard to understand how does it works, so i really want to know how create wav ormp3 file, how to concate\merge 2 or more files in one big, how to change speed of playing, how to set an addition filter to change sounds, set layouts of sound effects.. So can you help to me with it? How much it costs or size of donation i mustmake to get the guide\tutorial?

**Comment by philburk** [comment]
All of the things that you describe are general audio and signal processing. Those topics are not related to Oboe. Once you do all of the sound generation that you describe you would then use Oboe simply to move the sound from your app to the speakers. Oboe is not involved with synthesis or effects processing. There are lots of websites that will tell you how to do the audio processing. And that information works just as well whether you're on Android or iPhone Windows or Mac.
